### ISSUE TITLE
Upgrading HMRC fails in check-compatibility.js

### ISSUE THREAD

**Issue by glombek** [hmrc/hmrc-frontend#71]
Attempted to upgrade from 1.9.0 to 1.10.0 and the upgrade fails with the following message:

```
> hmrc-frontend@1.10.0 preinstall /tmp/build_3516378b2f360381fb12da449962d71b/node_modules/hmrc-frontend
> node check-compatibility.js
/tmp/build_3516378b2f360381fb12da449962d71b/node_modules/hmrc-frontend/check-compatibility.js:43
  const latestKnownHost = compatibility[installingHmrcVersion][host][0]
                                                              ^    
TypeError: Cannot read property 'prototype-kit' of undefined       
    at checkCompatibility (/tmp/build_3516378b2f360381fb12da449962d71b/node_modules/hmrc-frontend/check-compatibility.js:43:63)
    at Object.<anonymous> (/tmp/build_3516378b2f360381fb12da449962d71b/node_modules/hmrc-frontend/check-compatibility.js:61:65)
    at Module._compile (internal/modules/cjs/loader.js:1156:30)    
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)        
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47
```

I'm using the `govuk-prototype-kit` at version `9.6.0` which I believe is a supported option.

Workaround: change the name of the application in `package.json` to bypass the compatibility check.

**Comment by matthewmascord** [comment]
This should be fixed as of 1.10.1. Let me know if you have any issues.

**Comment by glombek** [comment]
Thank you, I didn't spot that update!
